packaging/installer/methods/packages.md

```diff
@@ -1,24 +1,54 @@
 # Install Netdata Using Native DEB/RPM Packages
 
+:::note
+
 Netdata provides pre-built native packages for most DEB- and RPM-based Linux distributions, following our [platform support policy](/docs/netdata-agent/versions-and-platforms.md).
 
-Our [kickstart.sh installer](/packaging/installer/methods/kickstart.md) uses these packages by default on supported platforms.
+:::
+
+Install Netdata using our [kickstart.sh installer](/packaging/installer/methods/kickstart.md), which automatically uses native packages on supported platforms.
 
-Add `--native-only` when running `kickstart.sh` to force native packages. The script will fail if native packages aren’t available.
+To ensure the installer uses only native packages, add `--native-only` when running `kickstart.sh`.
```

Comment:
```suggestion
To ensure the installer only uses native packages, add the `--native-only` option when running `kickstart.sh`.
```

Strictly speaking, both ‘uses only’ and ‘only uses’ are correct here, but ‘only uses’ will sound a bit more natural to a native speaker in this context.

Also, it’s a bit easier to understand the sentence as a whole if we explicitly clarify that `--native-only` is an option.